From 71be35ca7ede5d74e7ff444ce1b2e23728ed0221 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Thu, 4 Jun 2015 17:10:22 +0800
Subject: [PATCH] MLK-10795-2 ARM: dts: imx6ul: Add the clocks for CSI

Add the clocks for CSI in dts file
Add compatible string for imx6ul csi

Signed-off-by: Robby Cai <r63905@freescale.com>
(cherry picked from commit e4aecbd0937482228e1e6d67b6265c365f1c7cdd)
---
 arch/arm/boot/dts/imx6ul.dtsi | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index 5b71bba..dca3962 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -981,9 +981,13 @@
 			};
 
 			csi: csi@021c4000 {
-				compatible = "fsl,imx6ul-csi";
+				compatible = "fsl,imx6ul-csi", "fsl,imx6s-csi";
 				reg = <0x021c4000 0x4000>;
 				interrupts = <GIC_SPI 7 IRQ_TYPE_LEVEL_HIGH>;
+				clocks = <&clks IMX6UL_CLK_DUMMY>,
+					<&clks IMX6UL_CLK_CSI>,
+					<&clks IMX6UL_CLK_DUMMY>;
+				clock-names = "disp-axi", "csi_mclk", "disp_dcic";
 				status = "disabled";
 			};
 
-- 
1.8.0

